zeebe/broker/src/main/java/io/camunda/zeebe/broker/transport/partitionapi/InterPartitionCommandReceiverImpl.java

```diff
@@ -150,8 +157,9 @@ DecodedMessage decodeMessage(final byte[] message) {
       final var recordMetadata = new RecordMetadata();
       final var value = decodeCommand(messageDecoder, recordMetadata);
       decodeAuthInfo(messageDecoder, recordMetadata);
+      final var checkpointType = CheckpointType.valueOf(messageDecoder.checkpointType());
```

Comment:
:x: Once we have set a new protocol version and `sinceVersion`, we'll need to check whether this field is available yet or not.